src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -142,11 +147,11 @@ export const PresentationPanelHoverActions = ({
     const hoverActionsWidth =
       (rightHoverActionsRef.current?.offsetWidth ?? 0) +
       (dragHandleRef.current?.offsetWidth ?? 0) +
-      parseInt(euiThemeVars.euiSize, 10) * 2;
+      parseInt(euiTheme.size.m, 10) * 2;
```

Comment:
This should probably be `euiTheme.size.base` as `euiSize` was `16px` and `euiTheme.size.m` is `12px` 
(same for the following occurrences)

📖 [size docs](https://eui.elastic.co/#/theming/sizing/values), [migration overview](https://github.com/elastic/kibana/issues/199715#issuecomment-2546323208)